t/test-upgrades.bash

```diff
@@ -3,58 +3,58 @@
 set -o errexit -o noclobber -o nounset -o pipefail
 shopt -s failglob inherit_errexit
 
-UPGRADEABLE_VERSIONS="
+upgradeable_versions="
     2.5.2
     2.6.0
```

Comment:
There are a _lot_ of tags missing from that list. Please make sure to add them, ideally using the `git tag` method we've used elsewhere:

```shell
mapfile -t versions < <(
    git tag --list '[0-9]*.[0-9]*.[0-9]*' \
    | grep --fixed-strings --invert-match --line-regexp \
        --regexp=1.0.1 \
        --regexp=1.1.0 \
        --regexp=1.1.1 \
        --regexp=1.1.2 \
        --regexp=1.1.3 \
    | sort --version-sort
)
```

Reply:
its already on another commit (to come)